monad-logger/Control/Monad/Logger.hs

```diff
@@ -579,6 +582,22 @@ runStderrLoggingT = (`runLoggingT` defaultOutput stderr)
 runStdoutLoggingT :: MonadIO m => LoggingT m a -> m a
 runStdoutLoggingT = (`runLoggingT` defaultOutput stdout)
 
+-- | Run a block using a @MonadLogger@ instance which writes tuples to an unbounded channel.
```

Comment:
The only other `Chan` function I see is `withChannelLogger` which implements an unrelated useful behavior (something like a temporary limited bag of log items, which are either dropped or logged depending on whether an exception demands their inclusion).

I've committed some expanded documentation on the two new functions.

![screen shot 2016-01-07 at 14 44 39](https://cloud.githubusercontent.com/assets/1078772/12184967/3ced9b3c-b54d-11e5-8f64-af3f3ebaa03b.png)